ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -109,6 +109,7 @@
   private @Nullable String mFontFamily = null;
   private int mFontWeight = ReactTypefaceUtils.UNSET;
   private int mFontStyle = ReactTypefaceUtils.UNSET;
+  private int lengthFilterValue = UNSET;
```

Comment:
```suggestion
  private int mLengthFilterValue = UNSET;
```

Based on the previous code style, react-native using "m" for non public and non static field.